src/Kopernicus/Components/KSC.cs

```diff
@@ -38,83 +38,38 @@ namespace Kopernicus.Components
     public class KSC : SerializableMonoBehaviour
     {
         // PQSCity
-        [SerializeField]
         public Double? latitude;
-
-        [SerializeField]
         public Double? longitude;
-
-        [SerializeField]
         public Vector3? reorientInitialUp;
-
-        [SerializeField]
         public Vector3? repositionRadial;
-
-        [SerializeField]
         public Boolean? repositionToSphere;
-
-        [SerializeField]
         public Boolean? repositionToSphereSurface;
-
-        [SerializeField]
         public Boolean? repositionToSphereSurfaceAddHeight;
-
-        [SerializeField]
         public Boolean? reorientToSphere;
-
-        [SerializeField]
         public Double? repositionRadiusOffset;
-
-        [SerializeField]
         public Double? lodvisibleRangeMult;
-
-        [SerializeField]
         public Single? reorientFinalAngle;
 
         // PQSMod_MapDecalTangent
-        [SerializeField]
         public Vector3? position;
-
-        [SerializeField]
         public Double? radius;
-
-        [SerializeField]
         public Double? heightMapDeformity;
-
-        [SerializeField]
         public Double? absoluteOffset;
-
-        [SerializeField]
         public Boolean? absolute;
-
-        [SerializeField]
         public Double? decalLatitude;
-
-        [SerializeField]
         public Double? decalLongitude;
 
         // PQSCity Ground Material
-        [SerializeField]
         public Texture2D mainTexture;
-
-        [SerializeField]
         public Color? color;
 
         // Grass Material
-        [SerializeField]
         public GrassMaterial Material;
 
         // Editor Ground Material
-        [SerializeField]
         public Texture2D editorGroundTex;
-
-        [SerializeField]
         public Color? editorGroundColor;
-
-        [SerializeField]
         public Vector2? editorGroundTexScale;
-
-        [SerializeField]
         public Vector2? editorGroundTexOffset;
```

Comment:
Are we certain all those SerializeFields are unneeded?  I did not write this portion of the code, and am hesitant to remove them without a good explanation.

Reply:
Hey @R-T-B ! So one of the linters actually flagged those as redundant / invalid; I was as surprised as you. I ran into these kind of fields more years ago when I was first learning C#, so maybe they're not as necessary as they used to be?